src/app/[locale]/error.tsx

```diff
@@ -1,6 +1,6 @@
 'use client' // Error components must be Client Components
 
-import {IS_DEV_MODE} from '@enonic/nextjs-adapter/env';
+import {IS_DEV_MODE} from '@enonic/nextjs-adapter/common/env';
```

Comment:
We should probably think about this one before the final release, but for a beta I guess it's ok.